Source/WebCore/platform/graphics/avfoundation/cf/AVFoundationCFSoftLinking.h

```diff
@@ -37,14 +37,6 @@
 
 #include <d3d9.h>
 
-// Soft-link against AVFoundationCF functions and variables required by MediaPlayerPrivateAVFoundationCF.cpp.
-
-#ifdef DEBUG_ALL
-SOFT_LINK_DEBUG_LIBRARY(AVFoundationCF)
-#else
-SOFT_LINK_LIBRARY(AVFoundationCF)
-#endif
-
 // Functions
 
```

Comment:
I think you can delete this entire file.

Reply:
Thanks updated the commit; also found a few more files that I could remove too.